pkg/frontend/platformworkloadidentityrolesets_list.go

```diff
@@ -0,0 +1,45 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+)
+
+func (f *frontend) listPlatformWorkloadIdentityRoleSets(w http.ResponseWriter, r *http.Request) {
```

Comment:
:cry: well now I have to tell you to make this a `get`.

Reply:
Similar comment to some other places - I copy/pasted this from `pkg/frontend/openshiftversions_list.go` and that's how it turned out like this :sweat_smile:

This is another one that I'd like to leave as-is to avoid scope creep if that works for you, even though I agree with your idea.